modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -106,6 +106,14 @@ The Ingress Operator also converts the TLS `1.0` of an `Old` or `Custom` profile
 Ciphers and the minimum TLS version of the configured security profile are reflected in the `TLSProfile` status.
 ====
 
+|`clientTLS`
+|`clientTLS` authenticates client access to the cluster and services using mutual TLS authentication. If not set, then client TLS is not enabled.
+
+`clientTLS` has the required subfields, `spec.clientTLS.clientCertificatePolicy` and `spec.clientTLS.ClientCA`.
+
+The `ClientCertificatePolicy` subfield accepts one of the two values: `Required` or `Optional`. The `ClientCA` subfield specifies a ConfigMap that is in the openshift-config namespace. The ConfigMap should contain a CA certificate bundle.
+The `AllowedSubjectPatterns` specifies a list of regular expressions that are matched against the distinguished name on a valid client certificate to filter requests.  The regular expressions must use PCRE syntax. At least one pattern must match a client certificate's distinguished name; otherwise, the ingress controller rejects the certificate and denies the connection.
```

Comment:
We should probably make it clear that `AllowedSubjectPatterns` is optional; it is valid to leave it empty, in which case the ingress controller does not reject certificates based on the distinguished name.